packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -97,14 +175,7 @@ const CONTENT_NAME = 'ContextMenuContent';
 type ContextMenuContentOwnProps = Polymorphic.Merge<
   Omit<
     Polymorphic.OwnProps<typeof MenuPrimitive.Content>,
-    | 'trapFocus'
-    | 'disableOutsideScroll'
-    | 'portalled'
-    | 'onOpenAutoFocus'
-    | 'side'
-    | 'sideOffset'
-    | 'align'
-    | 'alignOffset'
```

Comment:
we're not removing these 2 anymore?

Reply:
We need them exposed for the submenu

Reply:
I think we've taken out `align` in `Menu` already and set it to `start` always so only `alignOffset` would remain I guess then.